README.rst

```diff
@@ -223,6 +223,33 @@ Odoo manifest files (``__manifest__.py`` or ``__openerp__.py``) and contain:
     in the addons directory, and external python dependencies.
   * ``python_requires``
 
+Installing Odoo CE and EE addons
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``setuptools-odoo`` has built-in knowledge of the addons that are part of the Odoo
+Community and Enterprise editions. Dependencies on these addons are condidered to be
+satisfied by the ``odoo`` distribution.
+
+This means Odoo must be *installed* in your python environment.
+
+As of Odoo 8 to 15, a good way to install Odoo is in a virtual environment. There are
+several possibilities, but the following bash commands should get you started:
+
+.. code:: console
+
+  $ python3 -m venv ./venv
+  $ source ./venv/bin/activate
+  (venv) $ python3 -m pip install --upgrade pip wheel
+  (venv) $ python3 -m pip install -r ./odoo/requirements.txt
```

Comment:
or
```
pip install https://github.com/odoo/odoo/archive/refs/heads/$VERSION.tar.gz
pip install -r https://raw.githubusercontent.com/odoo/odoo/$VERSION/requirements.txt
```

Reply:
@simahawk yes, but... no. Because you'll only get `base` in the addons path with this method. https://github.com/odoo/odoo/issues/16700 and https://github.com/odoo/odoo/pull/44001.

Reply:
I also "liked" #44001 but I completely forgot about it :rofl: :man_facepalming: